### ISSUE TITLE
Use of capital letters 

### ISSUE THREAD

**Issue by Studiesmurf** [ScrumFacilitators/EBM-Gids-NL#40]
In a Dutch translation, we should use capital letters when needed, not when being American.

Page 4:

E.g. "Strategisch Doelen" ( nevermind the typo, added that as a new issue), should be "strategische doelen".
Just like "Doelen Stellen" should be "Doelen stellen" as a title, and "Direct Tactisch Doel" should be "direct tactisch doel" (or "Direct tactisch doel").

**Comment by sjoerdly** [comment]
Hmm ook als Strategische Doelen een concept is in het framework? Een 'naam' eigenlijk?

Alternatief is om het met bold of cursief op te lossen.

**Comment by glaudia** [comment]
@sjoerdly goed idee mbt bold of cursief oplossen. Persoonlijk vind ik het ook irritant dat die woorden met hoofdletters worden geschreven in het nederlands en niet echt bevorderlijk voor de leesbaarheid

**Comment by sjoerdly** [comment]
Ja.  We moeten dan ook goed kijken naar het overige cursief of bold gebruik in de engelse gids. Dat het niet te verwarrend is. Underline wordt niet gebruikt in de engelse gids, dus dat is wellicht de beste optie.
